core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1319,6 +1319,15 @@ setting.preferences.autoConfirmEnabled=Enabled
 setting.preferences.autoConfirmRequiredConfirmations=Required confirmations
 setting.preferences.autoConfirmMaxTradeSize=Max. trade amount (BTC)
 setting.preferences.autoConfirmServiceAddresses=Monero Explorer URLs (uses Tor, except for localhost, LAN IP addresses, and *.local hostnames)
+setting.preferences.tradeLimits=Trade Limits
+setting.preferences.tradeLimitsEnabled=Increase Trade Limits
+setting.preferences.tradeLimitMax=Max. trade amount (BTC)
+setting.preferences.tradeLimitBlurb=Trading limits are set to reduce risk of costly disputes and losing funds for new \
+  users. Raise trading limits at your own risk. Every trader has their own pace, but it''s recommended to complete at \
+  least 5 trades before raising trade limits.\n\n\
+  By raising trading limits you acknowledge to be familiar with the basic use of Bisq, like backing up your data [HYPERLINK:https://bisq.wiki/Backing_up_application_data], \
+  trading rules [HYPERLINK:https://bisq.wiki/Trading_rules], dispute resolution [HYPERLINK:https://bisq.wiki/Dispute_resolution], penalties [HYPERLINK:https://bisq.wiki/Table_of_penalties] or find support [HYPERLINK:https://bisq.wiki/User_support].\n\n\
+  Account signing limits [HYPERLINK:https://bisq.wiki/Account_signing] are still applied to most National Account payment methods [HYPERLINK:https://bisq.wiki/Payment_methods], to reduce chargeback risk.
```

Comment:
[...] are still applied to buy BTC with most National Account payment [...]
Add **buy BTC** for clarity.